news/1023-change.md

```diff
@@ -0,0 +1,2 @@
+Fix RabbitMQ version check logic
+Tidy Java code
```

Comment:
just a note for future that the github markdown parser considers these to be one line, so when the changelog is generated they will be collapsed together

Reply:
Good point, thanks - I've switched it to a slightly more detailed sublist which might make more sense anyway. OK to merge now?

Reply:
good to go 👍